examples/src/main/java/com/hedera/hashgraph/sdk/examples/ConsensusPubSubChunkedExample.java

```diff
@@ -122,7 +113,7 @@ public static void main(String[] args) throws TimeoutException, PrecheckStatusEx
         // get the receipt to ensure there were no errors
         transaction.execute(client).getReceipt(client);
 
-        boolean largeMessageReceived = largeMessageLatch.await(60, TimeUnit.SECONDS);
+        boolean largeMessageReceived = largeMessageLatch.await(180, TimeUnit.SECONDS);
```

Comment:
This example can be flaky on localhost, i suggest running it at least 10 times to make sure it works

Reply:
I agree, I have following observations:
- when I start local-node it doesn't work
- when I re-run example 10x times, it worked each time

cc @agadzhalov @isavov wdyt?

Reply:
I think it's best to skip this test, as it might not even be designed to work on localhost.